src/engine/enginebuffer.cpp

```diff
@@ -705,6 +713,13 @@ void EngineBuffer::slotControlSlip(double v)
     m_slipEnabled = static_cast<int>(v > 0.0);
 }
 
+void EngineBuffer::slotControlSlipCancel(double v)
+{
+    if (m_slipEnabled.fetchAndAddAcquire(0)) {
```

Comment:
Oh sorry, I have misread that.  It is best to use load_atomic(m_slipEnabled) to avoid this.